src/components/molecules/VenueHero/VenueHero.tsx

```diff
@@ -1,49 +1,30 @@
 import React from "react";
 
-// Typings
-import { DetailsPreviewProps } from "./VenueHero.types";
-
-// Styles
 import * as S from "./VenueHero.styles";
-// import Button from "components/atoms/Button/Button";
-import { Link } from "react-router-dom";
-import { Button } from "react-bootstrap";
 
-const VenueHero: React.FC<DetailsPreviewProps> = ({
+export interface DetailsPreviewProps {
+  bannerImageUrl?: string;
+  logoImageUrl?: string;
+  name?: string;
+  subtitle?: string;
+  description?: string;
+  large?: boolean;
```

Comment:
For props like this, I personally wouldn't. `large` presents a better/nicer 'public API' than `isLarge`. When 'dereferencing' them in the function header though, it might make sense to 'rename' it (eg. `large: isLarge`) so that the rest of the 'internal code' follows our boolean naming standards.

Reply:
`large: isLarge` works for me